src/ceph-volume/ceph_volume/main.py

```diff
@@ -137,6 +142,11 @@ def main(self, argv):
         log.setup_console()
         logger = logging.getLogger(__name__)
         logger.info("Running command: ceph-volume %s %s", " ".join(main_args), " ".join(subcommand_args))
+
+        if args.version:
+            print('ceph version {0}'.format(ceph_volume.__version__))
+            return 0
```

Comment:
the entry point of the ceph-volume cli is `Volume.__init__()`, which return `None`. i think the same applies to `Volume.main()`.  probably we should just throw `SystemExit(0)` in this case ?

Reply:
returning `None` here in this case should be enough